src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -273,19 +273,19 @@
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.27.5" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.26.1" />
```

Comment:
We never released 4.26.1 - please use 4.25.2

Reply:
Thank you for catching that!